docs/content/concepts/sui-architecture/consensus.mdx

```diff
@@ -8,11 +8,34 @@ The basic purpose of consensus in blockchains is to agree on a consistent order
 
 On Sui, consensus has a simple API: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
 
-Byzantine-fault tolerant (BFT) consensus protocols are a rich area of research. The next-generation consensus engine set to roll out in Sui is based on the **Mysticeti** protocol.
-The protocol optimizes for both low latency and high throughput because:
-- It allows multiple validators to propose blocks in parallel, utilizing the full bandwidth of the network and providing censorship resistance. These are features of the DAG-based consensus protocols.
-- It takes only three rounds of messages to commit blocks from the DAGs, same as pBFT and matches the theoretical minimum.
+Sui uses the Mysticeti protocol to optimize for both low latency and high throughput. The benefits of Mysticeti include:
+
+- Supports multiple validators proposing blocks in parallel, utilizing the full bandwidth of the network and providing censorship resistance. These are features of the DAG-based consensus protocols.
+- The first consensus protocol to require only three rounds of messages to commit blocks from the DAGs (first of its kind), same as practical Byzantine Fault Tolerance and matches the theoretical minimum.
```

Comment:
CC @asonnino . Is this safe to be claimed here? I mean as the "first" one?

Reply:
I think I got this from the whitepaper, but it's really a no-value statement here so removing.